detection_rules/devtools.py

```diff
@@ -186,14 +216,60 @@ def git(*args, show_output=False):
 
         git("add", kibana_directory)
 
-        git("commit", "-S", "-m", message)
+        git("commit", "--no-verify", "-m", message)
         git("status", show_output=True)
 
+        if push:
+            git("push", "origin", branch_name)
+
         click.echo(f"Kibana repository {local_repo} prepped. Push changes when ready")
         click.secho(f"cd {local_repo}", bold=True)
 
+        return branch_name, long_commit_hash
+
     except subprocess.CalledProcessError as e:
-        client_error(e.returncode, e, ctx=ctx)
+        client_error(str(e), e, ctx=ctx)
+
+
+@dev_group.command("kibana-pr")
+@click.option("--token", required=True, default=get_github_token(), help="GitHub token to use for the PR")
```

Comment:
i think this is worth adding?
```suggestion
@click.option("--token", required=True, default=get_github_token(), help="GitHub token to use for the PR", password=True)
```